whack-a-landa/dom.js

```diff
@@ -69,57 +76,56 @@ function startGame() {
   // Hide nofication by the end of the game
   notification.classList.add('hidden');
   setTimeout(() => notification.style.opacity = 1, 400);
-  
-}
+};
 
 // Choose difficulty level (1 - AMATEUR - easy, 2 - BEAR JEW - noraml, 3 - ALDO RAIN - hard)
 function setLevel() {
   level = levels.options[levels.selectedIndex].value;
-  if (level == '1') {
-    minTime = 500;
-    maxTime = 1100;
-  }
-  if (level == '2') {
-    minTime = 200;
-    maxTime = 750;
-  }
-  if (level == '3') {
-    minTime = 100;
-    maxTime = 500;
-  }
-  levelName = level === '1' ? '(AMATEUR)' : level === '2' ? '(BEAR JEW)' : '(ALDO RAIN)';
-}
+  switch(level) {
+    case '1': minTime = 500;
+              maxTime = 1100;
+              levelName = '(AMATEUR)';
+    break;
+    case '2': minTime = 200;
+              maxTime = 750;
+              levelName = '(BEAR JEW)';
+    break;
+    case '3': minTime = 100;
+              maxTime = 500;
+              levelName = '(ALDO RAIN)';
+  };
+};
 
 // Open score board 
 function showScoreBoard() {
   scoreBoard.style.zIndex = 10;
   setTimeout(() => {
     scoreBoard.style.opacity = 1;    
   }, 100);
-}
+};
```

Comment:
please check where you need to put a semicolon, https://github.com/airbnb/javascript#semicolons

and let's use eslint with airbnb-base ruleset for the future in order to avoid such kind of mistakes